sound/soc/sof/pm.c

```diff
@@ -250,6 +250,22 @@ static int sof_suspend(struct device *dev, bool runtime_suspend)
 	return ret;
```

Comment:
@mrajwa No need to update the patch for this, but the git commit message a strange line-wrap to 50 characters. The recommend is to wrap at 72 characters or so.

Reply:
Well is there any written guide for that? I tend to use command line interface which is a fraction of the screen so 72 characters line is too much in my case and breaks the line.

Reply:
the recommendation is 50 chars for the first line (commit title) and 72 afterwards.

Checkpatch will tell you what's ok.

Reply:
https://github.com/torvalds/linux/pull/17#issuecomment-5661185